modules/rosa-sts-creating-a-cluster-with-customizations-ocm.adoc

```diff
@@ -0,0 +1,320 @@
+====
+
+. Select *Next*.
+
+. On the *Cluster update strategy* page, configure your update preferences:
+.. Choose a cluster update method:
+** Select *Individual updates* if you want to schedule each update individually. This is the default option.
+** Select *Recurring updates* to update your cluster on your preferred day and start time, when updates are available.
++
+[NOTE]
+====
+You can review the end-of-life dates in the update life cycle documentation for {product-title}. For more information, see _{product-title} update life cycle_.
+====
++
+.. If you opted for recurring updates, select a preferred day of the week and upgrade start time in UTC from the drop-down menus.
+.. Optional: You can set a grace period for *Node draining* during cluster upgrades. A *1 hour* grace period is set by default.
+.. Click *Next*.
++
+[NOTE]
+====
+In the event of critical security concerns that significantly impact the security or stability of a cluster, Red Hat Site Reliability Engineering (SRE) might schedule automatic updates to the latest z-stream version that is not impacted. The updates are applied within 48 hours after customer notifications are provided. For a description of the critical impact security rating, see link:https://access.redhat.com/security/updates/classification[Understanding Red Hat security ratings].
+====
+
+. Review the summary of your selections and click *Create cluster* to start the cluster installation.
+
+. If you opted to use *Manual* mode, create the cluster-specific Operator roles and OIDC provider manually to continue the installation:
+.. Under the *Installing cluster* section on the *Overview* page for your cluster, select *Installation waiting* -> *Review your options*.
+.. In the *Action required to continue the installation* dialog, select either the *AWS CloudFormation*, *AWS CLI*, or *ROSA CLI* tab and manually create the resources:
+** If you opted to use the *AWS CloudFormation* method, click the copy button next to the `aws cloudformation` commands and run them in the CLI.
+** If you opted to use the *AWS CLI* method, download the `Download.zip` file and extract the AWS CLI commands and policy files. Then, run the commands in the CLI.
+** If you opted to use the *ROSA CLI* method, click the copy button next to the `rosa create` commands and run them in the CLI.
+.. In the *Action required to continue the installation* dialog, click *x* to return to the *Overview* page for your cluster.
+.. Verify that the cluster *Status* in the *Details* section of the *Overview* page for your cluster has changed from *Installation pending* to *Installing*.
```

Comment:
Note that there is a delay before the status changes. (I'm not sure offhand exactly how long, maybe a minute or two?)

Reply:
Thank you. I have subsequently appended this to the step:

> There might be a short delay of approximately two minutes before the status changes.